src/routes/+layout.svelte

```diff
@@ -1,5 +1,36 @@
 <script lang="ts">
+  import type NavItemType from "$lib/components/Header/Nav/NavItemType";
+
+  import { navigating } from "$app/stores";
+  import { page } from "$app/stores";
   import "../app.scss";
+  import Banner from "$lib/components/landingPage/Banner.svelte";
+  import Footer from "$lib/components/landingPage/Footer.svelte";
+  import Header from "$lib/components/Header";
+  import Identifier from "$lib/components/landingPage/Identifier.svelte";
+
+  export let data;
+  const { navItems }: { navItems: NavItemType[] } = data;
+
+  // Update the active nav item based on the current path.
+  let activeNavItemIndex = -1;
+  $: {
+    activeNavItemIndex = navItems.findIndex((item) => item.link === $page.url.pathname);
+    navItems.forEach((_, i) => {
+      navItems[i].isCurrent = i === activeNavItemIndex;
+    });
+  }
```

Comment:
Since all the primary nav items are menus, this isn't doing anything but running extra loops right now. We can replace this with different logic if we decide we'll only use menus in the primary nav (you can also mark menus as current, as seen in [this USWDS example](https://federalist-3b6ba08e-0df4-44c9-ac73-6fc193b0e19c.sites.pages.cloud.gov/preview/uswds/uswds/develop/iframe.html?id=components-header--extended&args=&viewMode=story)).